V/VkFFTCUDALib/build_tarballs.jl

```diff
@@ -0,0 +1,87 @@
+using BinaryBuilder, Pkg
+using BinaryBuilderBase
+include(joinpath(@__DIR__, "..", "..", "fancy_toys.jl"))
+include(joinpath(@__DIR__, "..", "..", "platforms", "cuda.jl"))
+
+name = "VkFFTCUDALib"
+version = v"0.1.1"
+
+# Collection of sources required to complete build
+sources = [
+		   GitSource("https://github.com/PaulVirally/VkFFTCUDALib.git", "9c8cf2eae261b363e185e5043599ca2726f5aafd"),
+		   FileSource("http://us.download.nvidia.com/XFree86/Linux-x86_64/410.66/NVIDIA-Linux-x86_64-410.66.run", "8fb6ad857fa9a93307adf3f44f5decddd0bf8587a7ad66c6bfb33e07e4feb217"),
+		  ]
+
+script = raw"""
+# Setup some necessary CUDA annoyingness
+export CUDA_PATH="$prefix/cuda"
+ln -s $prefix/cuda/lib $prefix/cuda/lib64
+
+# Install CUDA driver libraries (all of this is to get libcuda.so to exist and work)
+mkdir -p ${prefix}/lib
+chmod +x NVIDIA-Linux-x86_64-*.run
+./NVIDIA-Linux-x86_64-*.run -x
+cp NVIDIA-Linux-x86_64-*/libcuda.so.* $prefix/lib/
+ln -s $(cd $prefix/lib; echo libcuda.so*) $prefix/lib/libcuda.so.1
+cp NVIDIA-Linux-x86_64-*/libnvidia-fatbinaryloader.so.* $prefix/lib/
```

Comment:
These files end up in the final tarball, but they shouldn't

Reply:
In 5ff08a8 I removed as many files in $workspace/lib as I could without getting warnings about broken symlinks.